draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -522,6 +522,11 @@ Note that some of the grammatical terms above reference documents that use
 different grammatical notations than this document (which uses ABNF from
 {{RFC5234}}).
 
+NOTE: The name of an attribute-value pair is not case sensitive (which differs
+from ABNF). So while they are presented here in CamelCase, such as "HttpOnly"
```

Comment:
I thought that AbNf strings were cafe insensitive, which is why RFC 7405 exists...

Reply:
Yep, you're right! My mistake, I must have misread it.